switchlink/sai/switchlink_handle_lag.c

```diff
@@ -357,6 +396,25 @@ void switchlink_create_lag_member(
   // lag member has already been created
   krnlmon_log_debug("Switchlink DB already has LAG Member config: %s",
                     lag_member_intf->ifname);
+
+  if (lag_member_intf->is_lacp_member) {
+    // check if lag member oper_state has changed
+    if (lag_member_info.oper_state != lag_member_intf->oper_state) {
+      status = set_lag_member_attribute(&lag_member_info,
+                                        lag_member_intf->oper_state);
+      if (status) {
+        krnlmon_log_error(
+            "newlink: Failed to update switchlink lag member: %s, error: %d\n",
+            lag_member_intf->ifname, status);
+        return;
+      }
+      // update the oper_state in the switchlink db
+      lag_member_info.oper_state = lag_member_intf->oper_state;
+      switchlink_db_update_lag_member_oper_state(&lag_member_info);
```

Comment:
handle an error case when this API returns entry not found.

Reply:
Addressed.